doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +273,48 @@ To build and program the source code from the command line, complete the followi
          west flash
 
    The device resets and runs the programmed sample or application.
+
+.. _thingy53_app_guide:
+
+Thingy:53 application guide
+***************************
+
+Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
+Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
+
+The samples and applications that comply with Thingy:53 aplication must comply with the following configuration:
```

Comment:
```suggestion
For a  sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
```

Also, neither of the below lists are steps that must be followed, so they can't be numbered lists. I recommend bullet points.